docs/source/host-collect-analyze/overview.md

```diff
@@ -0,0 +1,65 @@
+---
+title: Overview
+description: Host collectors and analyzers in support bundles
+---
+
+> New in v0.40.0 of Troubleshoot!
+
+## Introduction
+If you need to collect and analyze information that is not available when using in-cluster collectors, you can use host collectors to gather information about the environment, such as CPU, memory, available block devices, and so on. This is especially useful when you need to debug a Kubernetes cluster that is down.
+
+### Differences Between In-Cluster and Host Collectors
+[In-cluster collectors](https://troubleshoot.sh/collect/collectors), specified with the `collectors` property in the `SupportBundle` specification, collect information from a running Kubernetes cluster or schedule a resource in the cluster to dynamically generate data. 
+
+Host collectors gather information directly from the host that they are run on and do not have Kubernetes as a dependency. They can be used to test network connectivity, collect information about the operating system, and gather the output of provided commands.
+
+## Getting Started
+
+1. Download the support bundle binary from Github:
+
+```
+curl -L https://github.com/replicatedhq/troubleshoot/releases/download/v0.40.0/support-bundle_linux_amd64.tar.gz | tar xzvf -
+```
+
+**Note**: You can see the latest available releases at https://github.com/replicatedhq/troubleshoot/releases
+
+2. Create a YAML file using `kind: SupportBundle` and specify all of your host collectors and analyzers. You can use the following example as a test:
+
+```yaml
+apiVersion: troubleshoot.sh/v1beta2
+kind: SupportBundle
+metadata:
+  name: host-collectors
+spec:
+  hostCollectors:
+    - cpu: {}
+    - memory: {}
+  hostAnalyzers:
+    - cpu:
+        checkName: "Number of CPUs"
+        outcomes:
+          - fail:
+              when: "count < 2"
+              message: At least 2 CPU cores are required, and 4 CPU cores are recommended
+          - pass:
+              message: This server has at least 4 CPU cores
+    - memory:
+        checkName: "Amount of Memory"
+        outcomes:
+          - fail:
+              when: "< 4G"
+          - pass:
+              message: The system has at least 8G of memory
+```
+
+3. Generate the support bundle:
+
+```
+./support-bundle --interactive=false support-bundle.yaml
+```
+
+## Known Limitations and Considerations
+
+1. Although host collectors can technically be included in vendor support bundle specifications, host collectors are intended to run directly on the host and not using [KOTS](https://kots.io/). If host collectors run from KOTS, they are likely not to produce the desired result as they run in the context of the Kotsadm pod.
```

Comment:
Do you expect people will put host collectors and analyzers in a different spec than in-cluster ones, like Mark mentioned? If so, that might be a good call out somewhere on this overview page too. If I was using Troubleshoot and saw this new feature, I could definitely see myself starting out by adding them to the same spec and expecting this all to work. So while we do call out that running these from KOTS won't give you expected results, if we expect people to put them separately, we should say that.

If we don't expect them to be separate, how should people deal with the consequences of that?

Reply:
You could certainly have a spec where both would be included, but I mainly wanted to call out that Host Collectors are intended to be run from the CLI with the Support Bundle binary and pretty much never from KOTS(i.e Vendored).

Reply:
So I won't write off them being included in the same spec, but maybe there's a better way to make the KOTS vs. CLI distinction? Open to suggestions.

Reply:
Yeah, that's fair. What about saying "directly on the host using the CLI." That makes the connection more for me.